openapi.yaml

```diff
@@ -245,6 +245,13 @@ paths:
           schema:
             type: string
             example: 3f2d23ee214
+        - name: type
```

Comment:
Perhaps we could use `format` here instead of `type` so we're not overloading the word or using reserved Go keywords.

Reply:
I thought `format` could be confusing with file format i.e. json or txt. 
one option could be `filetype`

Reply:
I could see that, but this endpoint only retrieves the raw file. Getting a JSON representation of the file is a separate endpoint.

Reply:
updated `type` -> `format`